xemc3/core/load.py

```diff
@@ -1519,6 +1519,76 @@ def write_info_file(fn: str, ds: xr.Dataset) -> None:
             "TOTAL_RAD": dict(long_name="Total radiation", units="W"),
         },
     ),
+    "ENERGY_TRANSPORT": dict(
+        type="mapping",
+        vars={
+            "qconve": dict(
+                units="Wm^{-2}", long_name="Parallel electron convective heat flux"
+            ),
+            "qconvi": dict(
+                units="Wm^{-2}", long_name="Parallel ion convective heat flux"
+            ),
+            "qconde": dict(
+                units="Wm^{-2}", long_name="Parallel electron conductive heat flux"
+            ),
+            "qcondi": dict(
+                units="Wm^{-2}", long_name="Parallel ion conductive heat flux"
+            ),
+        },
+    ),
+    "ENERGY_BALANCE": dict(
+        type="mapping",
+        vars={
+            "div_qconve": dict(
+                units="Wm^{-3}",
+                long_name="Divergence of parallel electron convective heat flux",
+                scale=1e-6
```

Comment:
The scale is inverted, the value here is what is to be multiplied with the loaded field, so must be 1e6.